src/plugins/charts/server/plugin.ts

```diff
@@ -41,8 +41,18 @@ export class ChartsServerPlugin implements Plugin<object, object> {
         }),
         type: 'json',
         description: i18n.translate('charts.advancedSettings.visualization.colorMappingText', {
-          defaultMessage: 'Maps values to specified colors within visualizations',
```

Comment:
You should also delete the entries on the translations files in order to be translated again

Reply:
Hm... just interesting why CI is green =)

Reply:
oh... you are right,  didn't think about it

Reply:
CI fails only if there are entries on the translations that are not used, not if we change a translation unfortunately.

Reply:
done